src/mesh/impls/bout/boutmesh.cxx

```diff
@@ -170,6 +162,17 @@ int BoutMesh::load() {
   Mesh::get(jyseps2_2, "jyseps2_2", ny - 1);
   Mesh::get(ny_inner, "ny_inner", jyseps2_1);
 
+  // Set global grid sizes
+  GlobalNx = nx;
+  GlobalNy = ny + 2 * MYG;
+  if (jyseps1_2 != jyseps2_1) {
+    GlobalNy += 2*MYG;
+  }
+  GlobalNz = nz;
+
+  if (2 * MXG >= nx)
```

Comment:
I know this was already there, but it's identical to the check on L152, so let's delete it